test/system/950-preexec-hooks.bats

```diff
@@ -16,6 +16,8 @@ function teardown() {
 
 @test "podman preexec hook" {
     preexec_hook_dir=$PODMAN_TMPDIR/auth
+    preexec_hook_ok_file=/etc/containers/podman_preexec_hooks.txt
+    touch $preexec_hook_ok_file
```

Comment:
These tests are run on local developer machines, please do not touch system wide config directories from within the tests. The rm below is not executed when the test error out before that or when I manually cancel for whatever reason.
cc @edsantiago

Reply:
@edsantiago happy to change this if there's a better way forward.

Reply:
Whoa, I hadn't even looked at system tests because this is so hellaciously confusing. So let me get this straight: the /etc/whatever file does not actually contain anything, it's just a semaphore saying "if this file exists, then it's okay for podman to run preexec hooks"? Bleagh. That is going to cost us support time in the future.

Anyhow, yeah, the `touch` thing, you can't do that. First, all rootless tests are going to fail. You need to `skip_if_rootless`. Second, you need to do the `rm` inside `teardown()`. Which will not have your `$preexec_hook_ok_file` defined, so please define that in outer scope (like between the `load`s and `setup`).

Reply:
hahahahahahaha, well, I guess we don't test then.